docs/presets/pastel-rainbow.md

```diff
@@ -0,0 +1,21 @@
+[Return to Presets](./README.md#gruvbox-rainbow)
```

Comment:
this has to be your file path which is (./#pastel-rainbow)

Reply:
lol thanks, didn't see that.